### DISCUSSION TITLE
Just render few specific html raw elements in react markdown

### DISCUSSION THREAD

**Discussion by dante01yoon** [rehypejs/rehype#144] [Q&A] [answered]
Hi, among the html elements, I only want to render ul, li, ol, and p tag when text is given.

An other elements like `<button>`, `<select>` should be showed as normal text .

below is how my ReactMarkdown looks like, I want to use only few html tags to render under react markdown. 

disallowedElements no helps because it hide all the element specified .

```tsx
<ReactMarkdown
    disallowedElements={['input', 'textarea', 'script', 'select', 'form']}
    remarkPlugins={[remarkMath, remarkBreak, [remarkGfm, { singleTilde: false }]]}
    rehypePlugins={[rehypeKatex, [rehypeRaw, {options: ""}]]}
    sourcePos
    rawSourcePos
    includeElementIndex
    components={{


  // ...
/>

```

below dummy's p tag should be rendered.

```tsx
dummy = \`
  <p>hello 
 </p>
\`
```

below dummy texts should be treated as 'text', not to render 
```ts
dummy = \`
  <table>
 </table>
 <dog>
 <data>
\`
```

**Comment by wooorm** [comment]
Follow our support guide and use more words to explain your question: https://github.com/rehypejs/.github/blob/main/support.md.

**Reply by dante01yoon** [reply]
I updated my question. thanks

**Reply by wooorm** [reply]
Thanks. Can you please use markdown for code? this is very hard to read. https://commonmark.org/help/

**Reply by dante01yoon** [reply]
yeap I reformatted

**Comment by wooorm** [comment, marked as answer]
You can support HTML or you can *not* support HTML. There is no in between.

If you paste the markdown:

```
<table>
</table>
<dog>
<data>
```

On the website, you will see `<table> </table> <dog> <data>`. https://remarkjs.github.io/react-markdown/. If you want that, use that.

react-markdown is about markdown. Not HTML. Not an XML-like language. Markdown is a language with a spec. Markdown you use here on GH will work similarly to markdown that we support. And markdown in other places. You can’t make a new language with these projects. I think it’s much better if you explain to authors how to write markdown: https://commonmark.org/help/

**Reply by dante01yoon** [reply]
I see, thank you.
